interface/ceed-preconditioning.c

```diff
@@ -1753,6 +1753,77 @@ int CeedOperatorLinearAssemble(CeedOperator op, CeedVector values) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get the multiplicity of nodes across suboperators in a composite CeedOperator
+
+  Note: Calling this function asserts that setup is complete and sets the CeedOperator as immutable.
+
+  @param[in]  op               Composite CeedOperator
+  @param[in]  num_skip_indices Number of suboperators to skip
+  @param[in]  skip_indices     Array of indices of suboperators to skip
```

Comment:
I don't know if it's easy to say what to keep rather than what to skip, but we can change that later if it becomes an issue.